micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/ViewUrl/index.js

```
@@ -51,7 +82,7 @@ const ViewUrl = () => {
           <Button className="copyButton" variation={"secondary"} onClick={() => handleCopyUrl()} label={t("COPY_URL")}></Button>
         </div>
         <div className="sandbox-url-footer">{t("SANDBOX_URL_FOOT")}</div>
-        <Button onClick={onButtonClick} label={t("SIGN_IN")}></Button>
+        <Button isDisabled={buttonDisabled} onClick={onButtonClick} label={t("SIGN_IN")}></Button>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)